### DISCUSSION TITLE
Selected boot image not authenticate

### DISCUSSION THREAD

**Discussion by spalex810** [PartialVolume/shredos.x86_64#167] [Q&A] [answered]
While trying to boot shredos-2021.08.2_23_x86-64_0.34_20221231 from my USB stick on to an HP Elite Book, I am getting the message titled. Secure Boot: Selected boot image did not authenticate. Does anyone know of a work around for this? I am new to IT tasks and the field in general so any help would be appreciated(don't be afraid to talk to me like I'm 3)

**Comment by spalex810** [comment, marked as answer]
I disabled secure boot and then the computer allowed me to boot to shredos. And now I will continue wiping these laptops

**Reply by PartialVolume** [reply]
Glad you sorted it out, ShredOS will boot on legacy and UEFI hardware but as you found secure boot must be disabled.
